.github/workflows/ci.yaml

```diff
@@ -244,15 +239,24 @@ jobs:
       - name: Install Redis Operator
         run: |
           make deploy IMG=redis-operator:e2e
+          sleep 10
+          kubectl set env deployment/redis-operator-redis-operator -n redis-operator-system OPERATOR_IMAGE=redis-operator:e2e FEATURE_GATES=GenerateConfigInInitContainer=true
 
       - name: Wait for Redis Operator to be ready
         run: |
           kubectl wait --for=condition=available --timeout=300s deployment/redis-operator-redis-operator -n redis-operator-system
 
-      - name: Replace Redis Image Version
+      - name: Install k9s
```

Comment:
Why are we installing k9s

Reply:
it just for debug

Reply:
are you able to ssh into the node here ?

Reply:
yes, just as mxschmitt/action-tmate@v3 do